docs/CONTRIBUTION.md

```diff
@@ -0,0 +1,30 @@
+# Contributor Guide
```

Comment:
Note: the standard name for this file is `CONTRIBUTING.md`

Reply:
I am confusing... What is the change you want to have here?

Reply:
sorry, maybe https://github.com/canonical/solutions-engineering-automation/pull/165#discussion_r1988449220 makes it clearer. I'll resolve this thread and continue there.

Reply:
(the file should be named CONTRIBUTING.md, not CONTRIBUTION.md)